Changelog.md

```diff
@@ -6,6 +6,7 @@ Language Features:
 Compiler Features:
  * EWasm: Remove EWasm backend.
  * Parser: Introduce ``pragma experimental solidity``, which will enable an experimental language mode that in particular has no stability guarantees between non-breaking releases and is not suited for production use.
+ * Parser: Restrict experimental solidity to Constantinople and above.
```

Comment:
We can generally go easy on changelog entries for experimental solidity until it's actually user-facing.

I.e. I'd say no entries for now (we did it similarly while initially developing IR codegen).